siliconcompiler/report/streamlit_viewer.py

```diff
@@ -51,9 +52,13 @@
 
     chip = Chip(design='')
     chip.read_manifest(config["manifest"])
+    for file_path in config['comparison_chips']:
+        comparison_chip = Chip(design='')
+        comparison_chip.read_manifest(file_path)
+        chip.schema.cfg['history'][os.path.basename(file_path)] = comparison_chip.schema.cfg
```

Comment:
we seem to have lost the name a user might have provided

Reply:
At the moment, we do not let users supply names. Names are just 'chip1', 'chip2', ...

Reply:
lets discuss